.github/workflows/deploy.yaml

```
@@ -29,7 +29,7 @@ jobs:
 
       - name: Store build artifact
         uses: actions/upload-artifact@v4
-        if: github.ref == 'refs/heads/master'
+        if: github.ref == 'refs/heads/main'
```

How about not hardcoding it?

```suggestion
        if: github.ref_name == github.event.repository.default_branch
```

---

Better, thanks